librz/core/cmd_analysis.c

```diff
@@ -8004,6 +8004,8 @@ static int cmd_analysis_all(RzCore *core, const char *input) {
 			rz_cons_clear_line(1);
 			if (*input == 'a') { // "aaa"
 				bool experimental = input[1] == 'a';
+				ut64 file_offset = rz_config_get_i(core->config, "file.offset");
+				rz_analysis_add_io_registers_map(core->io, core->analysis, file_offset);
```

Comment:
move this within `rz_core_analysis_everything`

Reply:
I had done that in https://github.com/rizinorg/rizin/pull/1228/commits/227f9c3a606430469615d433359e6952edbcca70#diff-e4de03bacabad8d61bf51daee587cbf69970dae65a272486119cdaebfbf9e87aR7174, but from what I had seen, calling it from `rz_core_analysis_everything` broke a lot of analysis stuff and tests and I am still trying to find a reason.